.github/sync_upstream.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+echo "Enter hg"
+cd hg
+
+pushd openjfx/jfx-dev/rt/root
+echo "Update openjfx/jfx-dev/rt -> (root)"
+git hg fetch "http://hg.openjdk.java.net/openjfx/jfx-dev/rt"
+git hg pull "http://hg.openjdk.java.net/openjfx/jfx-dev/rt"
+popd
+
+echo "Exit hg"
+echo "Enter combined"
+
+cd ../combined
+
+echo "Check out master"
+git checkout master
+
+echo "Fetch (root)"
+git fetch "imports/openjfx/jfx-dev/rt/root"
+
+echo "Merge (root)"
+git merge "imports/openjfx/jfx-dev/rt/root/master" -m "Merge from (root)" --no-ff
```

Comment:
Probably adding the following should remove the extra commit,
    git fetch github master
    git merge github/master

    echo "Merge (root)"
    git merge "imports/openjfx/jfx-dev/rt/root/master"